src/rc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('rc')('DeltaChat', {})
```

Comment:
@Simon-Laux Just add some properties here with sane defaults for the logger.

Reply:
Ok, I will do that If you didn't already let's see

Reply:
Update: looks like we have to first fix the issue I mentioned here: https://github.com/deltachat/deltachat-desktop/pull/577#issuecomment-453229778